scripts/cleanup-project.sh

```
@@ -0,0 +1,143 @@
+    echo "   - Authentication is disabled"
+    echo "   - Sandboxing is disabled"
+    echo "   - Encryption at rest is disabled"
+fi
+
+# Create .gitignore if missing common entries
+echo "📄 Updating .gitignore..."
+cat << EOF >> .gitignore.tmp
+# Python
+__pycache__/
+*.py[cod]
+*$py.class
```

Shellcheck (0.10.0)

[warning] py is referenced but not assigned.

(SC2154)